### ISSUE TITLE
Missing epydoc link in twisted.web.http.getHost

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7247]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#7247|
|Type|enhancement|
|Created|2014-04-25 02:15:34Z|
|Branch|https://github.com/twisted/twisted/tree/missing-epydoc-in-http-7247|

There's a missing link to {{twisted.internet.tcp.Port}}.

<details><summary>Searchable metadata</summary>

```
trac-id__7247 7247
type__enhancement enhancement
reporter__Julian Julian
priority__normal normal
milestone__ 
branch__branches_missing_epydoc_in_http_7247 branches/missing-epydoc-in-http-7247
branch_author__julian julian
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1398392134000000 1398392134000000
changetime__1398393576000000 1398393576000000
version__None None
owner__Julian Julian
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="julian's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|julian commented|
|-|-|

(In [42547]) Branching to missing-epydoc-in-http-7247.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|Alex set owner to @Julian|
|-|-|

Kicked the buildbot. Assuming all tests pass, this LGTM, please land.

**Comment by twisted-trac** [comment]
|<img alt="julian's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|julian set status to `closed`|
|-|-|

(In [42550]) Merge missing-epydoc-in-http-7247: Fix a missing epydoc link.

Author: Julian
Reviewers: Alex
Fixes: [#7247](https://github.com/twisted/twisted/issues/7247)
